src/SonarScanner.MSBuild.Tasks/GetAnalyzerSettings.cs

```diff
@@ -192,27 +192,14 @@ bool IsTestAnalysisSupported()
 
             // See https://github.com/SonarSource/sonar-scanner-msbuild/issues/561
             // Legacy behaviour is to overwrite.
-            // The new (SQ 7.4+) behaviour is to merge only if sonar.[LANGUAGE].roslyn.ignoreIssues is false.
             var serverVersion = config?.FindServerVersion();
             if (serverVersion == null || serverVersion < new Version("7.4"))
```

Comment:
what's the server version for SonarCloud?

if it's not null, maybe open a ticket to remove this method? because it seems it'll always be true

we should evaluate the codebase and make a decision of removing the non supported LTS functionality

Reply:
The current version returned for SC is `8.0.0.29715`. So, indeed a version is returned for both SC and SQ. 

@tom-howlett-sonarsource we have here a functionality in the scanner that is only present with SQ version `<v7.4`. The behavior is that if you run the scanner against a SQ version that is `<v7.4` it will only embed our analyzers into the build and will remove all analyzers/source generators referenced in the project by the user. 
If I am not mistaken the 7.4 version was [released](https://www.sonarqube.org/sonarqube-7-4/) on October 29 in 2018. So, my question is if we could remove this functionality from the codebase?